docs/rule/no-curly-component-invocation.md

```diff
@@ -0,0 +1,94 @@
+## Use angle bracket syntax for components
+
+### Rule name: `no-curly-component-invocation`
+
+There are two ways to invoke a component in a template: curly compoment syntax (`{{my-component}}`), and angle bracket syntax (`<MyComponent />`). The difference between them is syntactical. You should favour angle bracket syntax as it improves readability of templates, i.e. disambiguates components from helpers, and is also the future direction Ember is going with Glimmer components.
```

Comment:
```suggestion
There are two ways to invoke a component in a template: curly compoment syntax (`{{my-component}}`), and angle bracket syntax (`<MyComponent />`). The difference between them is syntactical. You should favour angle bracket syntax as it improves readability of templates, i.e. disambiguates components from helpers, and is also the future direction Ember is going with the Octane Edition.
```

Just removing "Glimmer components" here, since they are orthogonal to using curly/angle invocation (you can invoke GC's with curlies).